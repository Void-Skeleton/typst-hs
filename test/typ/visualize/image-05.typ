// Test advanced SVG features.
#image("/assets/files/pattern.svg")

