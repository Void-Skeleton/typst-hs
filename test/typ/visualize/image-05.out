--- parse tree ---
[ Code
    "typ/visualize/image-05.typ"
    ( line 1 , column 2 )
    (Let
       (BasicBind (Just (Identifier "test")))
       (FuncExpr
          [ NormalParam (Identifier "x") , NormalParam (Identifier "y") ]
          (Block
             (CodeBlock
                [ If
                    [ ( Equals (Ident (Identifier "x")) (Ident (Identifier "y"))
                      , Block (Content [ Text "\9989" ])
                      )
                    , ( Literal (Boolean True)
                      , Block
                          (Content
                             [ Text "\10060"
                             , Text "("
                             , Code
                                 "typ/visualize/image-05.typ"
                                 ( line 1 , column 47 )
                                 (FuncCall
                                    (Ident (Identifier "repr"))
                                    [ NormalArg (Ident (Identifier "x")) ])
                             , Space
                             , Text "/"
                             , Text "="
                             , Space
                             , Code
                                 "typ/visualize/image-05.typ"
                                 ( line 1 , column 59 )
                                 (FuncCall
                                    (Ident (Identifier "repr"))
                                    [ NormalArg (Ident (Identifier "y")) ])
                             , Text ")"
                             ])
                      )
                    ]
                ]))))
, SoftBreak
, Comment
, Code
    "typ/visualize/image-05.typ"
    ( line 3 , column 2 )
    (FuncCall
       (Ident (Identifier "image"))
       [ NormalArg (Literal (String "/assets/files/pattern.svg")) ])
, ParBreak
]
--- evaluated ---
document(body: { text(body: [
]), 
                 image(path: "/assets/files/pattern.svg"), 
                 parbreak() })
